### ISSUE TITLE
🐛 BUG: Unable to Navigate to Search When Screen Width is <1020px

### ISSUE THREAD

**Issue by Tevingatheru** [dailydotdev/daily#1156]
### What went wrong? 🤔

I am unable to access the search page via new tab then click Search navigation option.
When I Open a new tab I am navigated to daily.dev home page(as expected). When I click on `Search` tab I am directed to `chrome-extension://jlmpjdjjbgclbocgajdjefcidcncaied/search`
![image](https://github.com/dailydotdev/daily/assets/17544181/69ca0032-2c1d-48c0-bb78-448d7d3c385d)

Work around
1. Opening new tab
2. Click on any Profile.
3. Then click on search tab.

### Expected Behavior

1. Open browser.
2. Open new tab.(if you are not already on a new tab)
3. See daily dev home page.
4. Click on Search navigation option.
5. See Search Page.

### Steps to Reproduce Issue

```bash
1. Open browser.
2. Open new tab.(if you are not already on a new tab)
4. See daily dev home page.
3. Adjust width to less than 1020 px.
4. Click on Search navigation option.
5. See ERR_FILE_NOT_FOUND page.
```

### Solution Proposed

1. Fix url redirect to point to correct file location.
2. Make sure that regardless of screen size the feature is the same. I suggest using the button as it allow for more flexibility. 

### Screenshots

```bash
![Error page](https://github.com/dailydotdev/daily/assets/17544181/69ca0032-2c1d-48c0-bb78-448d7d3c385d)
```

### Environment

Windows 10, Brave

### Browsers

Brave

### OS

Windows

### Additional Context

```bash
The issue seems to do with the different approach taken with different screen widths. If the width is less than 1020px then the search option uses <a href='/search'> tag which point to '/search'. However if the width of the screen is greater than 1020px (>1020px) then the search option is a button using <button> tag. Thus I think the cause of this issue is having different features based on screen width.

This also affects the Squads navigation option.
```

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by idoshamun** [comment]
Hi @Tevingatheru,

Thanks for the report. We will look into it

**Comment by capJavert** [comment]
This + other links in main navigation were fixed in https://github.com/dailydotdev/apps/pull/3088. It is gonna take some time to release new extension, but in the next version this fix will land.

**Comment by rebelchris** [comment]
It's now available in the new extension version 3.31.12 and higher.
Thanks for reporting this 🙏
